setup.sh

```diff
@@ -35,7 +27,12 @@ clean(){
 install(){
   echo 'Installing project dependencies...'
   npm install
-  bower install --config.interactive=false
+  if [ -f $SHRINKWRAP ]; then
```

Comment:
Weird, I wonder why...there's gotta be something else missing. No idea if you have the capacity, but I'd suggest figuring that out because a new dev might not know to update a version here and package.json